contracts/src/v0.8/functions/dev/v1_X/interfaces/IFunctionsBilling.sol

```diff
@@ -53,9 +62,12 @@ struct FunctionsBillingConfig {
   uint32 feedStalenessSeconds; //                  ║ How long before we consider the feed price to be stale and fallback to fallbackNativePerUnitLink.
   uint32 gasOverheadBeforeCallback; //             ║ Represents the average gas execution cost before the fulfillment callback. This amount is always billed for every request.
   uint32 gasOverheadAfterCallback; //              ║ Represents the average gas execution cost after the fulfillment callback. This amount is always billed for every request.
-  uint72 donFee; //                                ║ Additional flat fee (in Juels of LINK) that will be split between Node Operators. Max value is 2^80 - 1 == 1.2m LINK.
+  uint72 donFee; //                                ║ Additional flat fee (denominated in cents of USD, paid as LINK) that will be split between Node Operators.
```

Comment:
If this is cents you can reduce the type by a lot! At most uint32 and potentially uint16